hyperspace/cosmos/src/lib.rs

```diff
@@ -23,11 +23,13 @@ pub mod events;
 pub mod key_provider;
 pub mod light_client;
 pub mod provider;
-#[cfg(any(test, feature = "testing"))]
-pub mod test_provider;
+
+// #[cfg(any(test, feature = "testing"))]
+// pub mod cosmos_test_provider;
```

Comment:
Not needed anymore, I guess

Reply:
The ``test_provider`` makes the unit test not working. The comment thing is only for testing purpose. Let me uncomment it.